internal/pkg/service/stream/storage/level/local/writer/format/csv/csv.go

```diff
@@ -0,0 +1,48 @@
+package csv
+
+import (
+	"io"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/service/stream/mapping/table/column"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/stream/storage/level/local/writer"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/stream/storage/level/local/writer/format/csv/fastcsv"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/stream/storage/model"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/errors"
+)
+
+type Writer struct {
+	columns column.Columns
+	pool    *fastcsv.WritersPool
+}
+
+// NewWriter creates CSV writers pool and implements format.FormatWriter
+// The order of the lines is not preserved, because we use the writers pool,
```

Comment:
Isn't the problem that we do not preserve the order? However as it is streamed, it is expected that the messages could be delayed therefore the order does not matter. Am I right?